lib/coverband/adapters/file_store.rb

```diff
@@ -46,6 +42,10 @@ def covered_lines_for_file(file)
 
       private
 
+      def save_coverage(report)
+        File.open(path, 'w') { |f| f.write(report.to_json) }
```

Comment:
By default use file system to persist report?

Reply:
so this is only for the file store which mostly has been used for testing and development it would work fine for single server deployments but falls over for anything with more than one server so Redis is the primary adapter... but with the new pattern I think it would be simple to support memcache, ActiveRecord, and S3 as a store... I think for S3 just logging all the files and doing the combinational math to remove duplicate counts only when reporting

Reply:
Oh gotcha. I think i misread this before. Makes sense.